libs/web/feature-board/src/lib/components/NewGoalForm.tsx

```diff
@@ -96,8 +96,8 @@ export const NewGoalForm = () => {
   return (
     <Box
       sx={{
-        backgroundColor: 'white', // TODO - replace with theme
-        color: 'primary.contrastText',
+        backgroundColor: 'color.white', // TODO - replace with theme
+        color: 'color',
```

Comment:
this did not need to change, it could stay primary.contrastText, no reason to change it to `color`, which points at an object and not a keypair value.